static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.js

```diff
@@ -56,6 +58,7 @@ import {
 import { getAppNexusDirectBidParams } from './appnexus';
 
 // The below line is needed for page skins to show
+// Now it's a Promise, is this needed? 2020-10-23
```

Comment:
probably not - I bet this just kicked of `onConsentChange` - def worth checking after this is merged

Reply:
That's @ioanna and I's thought exactly.